trade-finance-manager-ui/server/controllers/case/amendments/amendmentBankReviewDate.get.api-test.js

```diff
@@ -231,9 +264,9 @@ describe('amendmentBankReviewDate routes', () => {
           status: 200,
           data: MOCK_AMENDMENT_COVERENDDATE_CHANGE_USING_BANK_REVIEW_DATE,
         });
-        api.getFacility = jest
-          .fn()
-          .mockResolvedValueOnce({ facilitySnapshot: { dates: { isUsingFacilityEndDate: false, bankReviewDate: new Date(2025, 11, 11).toISOString() } } });
+        api.getFacility = jest.fn().mockResolvedValueOnce({
+          facilitySnapshot: { ...gefFacility.facilitySnapshot, dates: { isUsingFacilityEndDate: false, bankReviewDate: new Date(2025, 11, 11).toISOString() } },
```

Comment:
could this be a const object and the object be added to jest.fn().mockResolvedValueOnce(object);

Reply:
Sure, have changed